code/to_po.py

```diff
@@ -15,16 +16,34 @@
 xmi_doc = xmi_document(fn)
 bfn = os.path.basename(fn)
 
-all_names = re.compile("\\b(%s)\\b" % "|".join(sorted((item.name for item in xmi_doc if item.type != "FUNCTION"), key=lambda s: -len(s))))
+items = list(xmi_doc)
+names = ["USERDEFINED", "NOTDEFINED"]  # adding those two to be translatable values, as they occur in descriptions but will not be listed in bSDD. 
+for item in items:
+    if item.type != "FUNCTION":
+        names.append(item.name)
+    # add all enumeration values except two 
+    if item.type in ("ENUM", "PENUM"):
+        for c in item.children:
+            if c.name not in ("USERDEFINED", "NOTDEFINED", "UNSET"):
+                if len(c.name) > 1 and not c.name.isnumeric():
```

Comment:
What are examples of cases violating this?

Reply:
many values like "A", "B", "10", "1234" etc.